misc/install-deps.sh

```diff
@@ -16,30 +52,14 @@ if [ ! -f /etc/os-release ]; then
 fi
 
 distro=$(grep "^ID=" /etc/os-release | cut -d\= -f2 | sed -e 's/"//g')
+id_like=$(grep "^ID_LIKE=" /etc/os-release | cut -d\= -f2 | sed -e 's/"//g')
+
+install_packages "$distro"
+
+for distro in $id_like; do
+    install_packages "$distro"
+done
 
-case $distro in
-    "ubuntu" | "debian" | "raspbian" | "kali" | "linuxmint")
-        apt-get $OPT install pandoc libdw-dev python3-dev libncursesw5-dev pkg-config
-        apt-get $OPT install libluajit-5.1-dev || true
-        apt-get $OPT install libcapstone-dev || true ;;
-    "fedora")
-        dnf install $OPT pandoc elfutils-devel python3-devel ncurses-devel pkgconf-pkg-config
-        dnf install $OPT luajit-devel || true
-        dnf install $OPT capstone-devel || true ;;
-    "rhel" | "centos")
-        rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-        yum install $OPT pandoc elfutils-devel python3-devel ncurses-devel pkgconfig
-        yum install $OPT luajit-devel || true
-        yum install $OPT capstone-devel || true ;;
-    "arch" | "manjaro")
-        pacman $OPT -S pandoc libelf python3 ncurses pkgconf
-        pacman $OPT -S luajit || true
-        pacman $OPT -S capstone || true ;;
-    "alpine")
-        apk $OPT add elfutils-dev python3-dev ncurses-dev pkgconf
-        apk $OPT add luajit-dev || true
-        apk $OPT add capstone-dev || true ;;
-    *) # we can add more install command for each distros.
-        echo "\"$distro\" is not supported distro, so please install packages manually."
-        echo ;;
-esac
+echo "\"$distro\" is not supported distro, so please install packages manually."
```

Comment:
I think the `$distro` should have the value from the original `ID` field and not sure if the for loop above keeps it.  How about using a different variable for `ID_LIKE`?

Reply:
Sure. I just changed the variable name so `$distro` should only have value from the original `ID`.